[Log]OutLog Start
[Exception]LuaException: Scene/Fight/FightScene:45: attempt to perform arithmetic on local 'place' (a nil value)
stack traceback:
	Scene/Fight/FightScene:45: in function 'GetPos'
	Data/CardInfo:31: in function 'SetPlace'
	Data/CardInfo:25: in function 'ctor'
	Common/functions:169: in function 'new'
	Scene/Fight/FightScene:40: in function 'InitMonster'
	Scene/Fight/FightScene:18: in function 'Init'
	Main.lua:27: in function <Main.lua:16>
[Log]OutLog Finish
